ui/templates.go

```diff
@@ -9,7 +11,7 @@
     </head>
     <body>
         <!-- Your source after making -->
-        <script src="/script.js"></script>
-        <script>Elm.Main.embed(document.body)</script>
+        <script src="{{ .ExternalURL }}/script.js"></script>
```

Comment:
Is this still needed? In an k8s cluster with `kubectl proxy` `script.js` is loaded properly without specifying the `-web.external-url` parameter.